src/dpw.jl

```diff
@@ -96,7 +83,7 @@ function simulate(dpw::DPWPlanner, snode::Int, d::Int)
     dpw.reset_callback(dpw.mdp, s) # Optional: used to reset/reinitialize MDP to a given state.
     if isterminal(dpw.mdp, s)
         return 0.0
-    elseif d == 0
+    elseif d == 0 || (timeout_us > 0.0 && CPUtime_us() > timeout_us)
```

Comment:
This seems redundant with the timeout check in the search loop and unrelated to the main PR subject. Why was it added?